UnitsNet/CustomCode/Quantities/Density.extra.cs

```diff
@@ -7,26 +7,6 @@ namespace UnitsNet
 {
     public partial struct Density
     {
-        /// <summary>
-        ///     Gets <see cref="Molarity" /> from this <see cref="Density" />.
-        /// </summary>
-        /// <param name="molecularWeight"></param>
-        public Molarity ToMolarity(Mass molecularWeight)
```

Comment:
1. We can't remove these without breaking backwards compatibility. At best, we can mark with `[Obsolete("some text")]` and remove in #563 .
2. Since Density should continue to exist, shouldn't these methods keep existing too or do we want people to use `MassConcentration` instead?

Reply:
My bad- I've re-introduced the methods/operators (marking them as Obsolete)